### PR TITLE
Update signing-libraries.md

### PR DESCRIPTION
**Pull request by Blits-Labs**
Update signing-libraries.md to include:

[filecoin-js-signer](https://github.com/blitslabs/filecoin-js-signer), a library written in pure Typescript/Javascript to create, sign & broadcast messages to interact with Filecoin's built-in actors, like Payment Channel and Multisig. 

[Blake2b-256 Solidity](https://github.com/blitslabs/filecoin-blake2b-solidity), an implementation of the Blake2b-256 hash function used by Filecoin, that can be used to create interoperability with EVM-compatible blockchains, and facilitate the creation of the HTLCs. 

Both libraries are currently being used by the [Filecoin Loans](https://github.com/blitslabs/filecoin.loans-monorepo) project, which received a Grant by the Filecoin Foundation.

### CODE REVIEW

**Comment by mishmosh** [comment]
Thanks for the PR @Blits-Labs! @arajasek Can you confirm that these 2 signing libraries are ready for prime-time, or reassign to someone who can?

File: docs/build/signing-libraries.md [hunk 1]
```diff
@@ -20,6 +20,15 @@ The [Filecoin signing tools project](https://zondax.ch/projects/filecoin-signing
 
 The [source code repository can be found here](https://github.com/Zondax/filecoin-signing-tools).
 
+## Filecoin JS Signer
+
+[Filecoin JS Signer](https://github.com/blitslabs/filecoin-js-signer) is a signing library and RPC client written in pure Typescript / Javascript with the necessary methods to create, sign and broadcast messages to send FIL, and interact with Filecoin's built-in actors (like Payment Channel and Multisig), among other utility functions.
```
Comment (johnnymatthews) [inline docs/build/signing-libraries.md:25]: > Typescript / Javascript

Is it fair to say that this project is written in _just_ Typescript?

File: docs/build/signing-libraries.md [hunk 2]
```diff
@@ -20,6 +20,15 @@ The [Filecoin signing tools project](https://zondax.ch/projects/filecoin-signing
...
+
 ## Filecoin.js
 
 [Filecoin.js](https://filecoin-shipyard.github.io/filecoin.js/) aims to be a complete library for interacting with local or remote Filecoin nodes. It can be used to [sign](https://filecoin-shipyard.github.io/filecoin.js/docs/sign-message) and verify messages and supports multiple wallet providers.
+
+## Blake2b-256 Solidity
+Solidity implementation of the BLAKE2b-256 hash function used by Filecoin that can be used to create interoperability with EVM-compatible blockchains and facilitate the creation of Hash Time Locked Contracts.
+
+The [source code repository can be found here](https://github.com/blitslabs/filecoin-blake2b-solidity).
```
Comment (johnnymatthews) [inline docs/build/signing-libraries.md:34]:
```suggestion
## Blake2b-256 Solidity

Solidity implementation of the BLAKE2b-256 hash function used by Filecoin that can be used to create interoperability with Ethereum-compatible blockchains and facilitate the creation of hash time-locked contracts.

The [source code repository can be found here](https://github.com/blitslabs/filecoin-blake2b-solidity).
```

**Review by johnnymatthews — APPROVED** [review]

**Comment by johnnymatthews** [comment]
This is good to merge once we get approval from @arajasek.

**Comment by johnnymatthews** [comment]
Closing as stale.

**Pull request closed without merging**